sys-fs/genfstab/genfstab-28-r1.ebuild

```diff
@@ -1,12 +1,14 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
+inherit bash-completion-r1
+
 DESCRIPTION="Genfstab - generate output suitable for addition to an fstab file"
-HOMEPAGE="https://github.com/scardracs/genfstab https://man.archlinux.org/man/genfstab.8"
-SRC_URI="https://github.com/scardracs/genfstab/releases/download/${PV}/${P}.tar.gz"
-S="${WORKDIR}"
+HOMEPAGE="https://github.com/archlinux/arch-install-scripts https://man.archlinux.org/man/genfstab.8"
+SRC_URI="https://github.com/archlinux/arch-install-scripts/archive/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/arch-install-scripts-tags-v${PV}"
```

Comment:
curious on how $P expands to "arch-install-scripts-tags-v${PV}". Did it myself and used only `S="${WORKDIR}/arch-install-scripts-${PV}"`

Reply:
`S="${WORKDIR}/arch-install-scripts-tags-v${PV}"` when `SRC_URI` is `https://github.com/archlinux/arch-install-scripts/archive/tags/v28.tar.gz` , and
`S="${WORKDIR}/arch-install-scripts-${PV}"` when `SRC_URI` is `https://github.com/archlinux/arch-install-scripts/archive/refs/tags/v28.tar.gz` ,

Their content is the same.

Reply:
oh didn't notice the /refs/, mb